### ISSUE TITLE
Time zone test failure

### ISSUE THREAD

**Issue by jduss4** [open-oni/open-oni#386]
When running the tests on the current dev branch (b30527cda3936d58da1cab41ba9df744575106f2) I'm seeing an awful lot of errors that appear to be related to the time zone.  The test is expecting -04:00 but is getting back +00:00

```
                     u'metadata': [{u'label': u'Ingested',
-                                   u'value': u'2009-03-26T20:59:28+00:00'},

+                                   u'value': u'2009-03-26T20:59:28-04:00'},
```

There are also a lot of warnings:

```
RuntimeWarning: DateTimeField Title.version received a naive datetime (2019-02-05 16:39:50.715336) while time zone support is active.
```

You can get rid of these warnings (but not the errors) by putting this is the `test_settings.py` file:

```
USE_TZ = False
```

I think we need to look into what is going on here and how we can use timezone aware date objects where possible!

I'm guessing it might be related to updating django versions, because I'm also seeing this failed test

```
======================================================================
FAIL: test_django_version (core.tests.test_system.SystemTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/openoni/core/tests/test_system.py", line 10, in test_django_version
    self.assert_(django.get_version() >= '1.2.1')
AssertionError: False is not true
```

**Comment by techgique** [comment]
There were some changes in time zone handling in the Django updates. In addition (I can't remember off the top of my head for sure), the old settings files _may_ have omitted or differed from the `USE_TZ = True` and `TIME_ZONE = 'UTC'` which are included from the [defaults Django generates for a new app](https://github.com/open-oni/open-oni/blob/b30527cda3936d58da1cab41ba9df744575106f2/onisite/django_defaults.py). I don't think I touched the test settings though. Perhaps they should be brought more in line with the other settings.

I haven't looked into these tests, but I recall my thought pattern being that we don't display any times after dates in our pages, and it didn't appear to change the dates. I just spot checked a few dates by comparing to the dates in the actual page images. So I just moved forward assuming we would return to fixing tests later when we're nearing a time we'll merge back into `master`.
